utils/node_local_test.py

```diff
@@ -1904,10 +1957,8 @@ def _run_test():
                 break
             pt.call_index = pt.call_index + 1
 
-    pools = get_pool_list(server)
-    while len(pools) < 1:
-        pools = make_pool(server)
-    pool = pools[0]
+    server.get_test_pool()
+    pool = server.test_pool
```

Comment:
(very minor) would it make sense to name this pobj instead of pool, 
only to distinguish it from the many other cases where there exists
pool = server.get_test_pool() in which pool is the pool UUID?

Reply:
Probably, I have changed one of the types in this PR from a string representing a uuid to a object where you need to fetch the uuid, however this code isn't imported from elsewhere so we do not need to worry about compatibility, and we can be sure from testing/static analysis that we've got all places.